From b24d7df65eb62a0338ab2359e5127e48756e9ee4 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Thu, 21 Apr 2022 11:36:35 +0800
Subject: [PATCH 044/101] arm64: dts: rockchip: disable usb3 on quartz64

USB 3 ports on Quartz64 is of bad quality as it shares lines with SATA.
Disable usb3 support to prevent issues.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
Ref: Contact the author.
---
 arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
index 37a1303d9a34f..dac1b96ec9744 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-quartz64-a.dts
@@ -790,6 +790,10 @@ &usb_host0_xhci {
 
 /* usb3 controller is muxed with sata1 */
 &usb_host1_xhci {
+	phys = <&usb2phy1_otg>;
+	phy-names = "usb2-phy";
+	extcon = <&usb2phy1>;
+	maximum-speed = "high-speed";
 	status = "okay";
 };
 
@@ -803,7 +807,7 @@ &usb2phy0_host {
 };
 
 &usb2phy0_otg {
-	phy-supply = <&vcc5v0_usb20_otg>;
+	phy-supply = <&vcc5v0_usb20_host>;
 	status = "okay";
 };
 
-- 
2.46.0

